### DISCUSSION TITLE
How can Vedo write all actors with write()?

### DISCUSSION THREAD

**Discussion by ZiguoAtGitHub** [marcomusy/vedo#926] [Q&A] [answered]
Hi Marco,
I want to use write() function in vedo.file_io module to generate 3d-party models with other file formats.
I tried it but only one actor was outputted. 
How to generate the model file (i.e. obj format) with all actors in the scene?
Thank you!

**Comment by ZiguoAtGitHub** [comment]
Maybe vtkOBJExporter is more suitable.😃

**Reply by marcomusy** [reply]
Hi - that is not implemented but in principle it should not be hard to add it to `file_io.export_window()`.

**Reply by ZiguoAtGitHub** [reply, marked as answer]
Yes.
I had solved it by a rude way: writing every actor to separate obj file and combining them.😂
BTW, the file generated with vtkOBJExporter could not be imported by Rhino.
